### PR TITLE
update ConfigStore API doc

### PR DESCRIPTION
**Pull request by jieru-hu**
changes includes:
1. move up configstore API doc in the tutorial, give an intro before we start to interact with it. 
2. Add an example on ConfigStore and YAML input work side by side.
3. some minor fix in other parts of the tutorial

[1] 
image

### CODE REVIEW

**Comment by jieru-hu** [comment]
@tangbinh @hudeven it'd be good to get your feedback on the ConfigStore API doc :) 

A deployed version of the changes in this PR can be found here https://deploy-preview-2004--hydra-preview.netlify.app/docs/tutorials/structured_config/config_store/

**Comment by tangbinh** [comment]
@jieru-hu Thank you for adding examples to the doc. These examples are certainly helpful, but I wonder if it possible that we make changes to the API itself. In particular, do you think we can let users interact with `ConfigStore` by including some `get` methods?

It's really hard to debug `ConfigStore` issues at the moment because it's functioning like a blackbox. Some users have encountered name conflicts that were silenced by `ConfigStore`, yet they couldn't figure out which config actually got registered.

**Comment by jieru-hu** [comment]
@tangbinh thanks! per our offline discussions, the improvements to the API will come later. This PR is to add more context for the API in the doc.

File: website/docs/tutorials/structured_config/10_config_store.md [hunk 1]
```diff
@@ -33,6 +33,109 @@ class ConfigStore(metaclass=Singleton):
...
+def my_app(cfg: DictConfig) -> None:
+    print(OmegaConf.to_yaml(cfg))
+
+
+if __name__ == "__main__":
+    my_app()
+```
+
+
+Awesome, now our application has access to both `db` config group options, let's run the application to verify:
```
Comment (pixelb) [inline website/docs/tutorials/structured_config/10_config_store.md:112]: Grammar nit:

s/Awesome, now our/Now that our/

**Review by pixelb — APPROVED** [review]

**Pull request merged**